xls/dslx/frontend/ast.h

```diff
@@ -2282,6 +2283,8 @@ class Function : public AstNode {
     disable_format_ = disable_format;
   }
   bool disable_format() const { return disable_format_; }
+  void set_used_in_tests(bool used_in_tests) { used_in_tests_ = used_in_tests; }
```

Comment:
I think this is not "used in tests" conceptually more like "test only"

Reply:
Yes, you're right. I changed that